cli/pack/options.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  globs: [
+    'globs',
+    {
+      array: true,
+      alias: ['g'],
+      default: ['./components/**/client.js'],
```

Comment:
i think we're introducing a regression, in which components client.js is being shipped to the browser in edit mode (which is a breaking change relative to existing behavior as documented [here](https://github.com/clay/clay-kiln/blob/2adb7bb602a31928c30fb153360e9dedd08973a4/docs/component.md#clientjs)

i'm not entirely sure where the appropriate place to make these changes is, but seeing this jogged my memory

Reply:
Thanks for flagging this. I shoehorned a fix in with [`5b1ad42` (#87)](https://github.com/clay/amphora-html/pull/87/commits/5b1ad42e80de21d619d4c96f68c24b11c39cb732).